src/app/ClioApplication.cpp

```diff
@@ -103,6 +104,16 @@ ClioApplication::run(bool const useNgWebServer)
     // Interface to the database
     auto backend = data::makeBackend(config_);
 
+    {
+        auto const migrationInspector = migration::makeMigrationInspector(config_, backend);
+        // Check if any migration is blocking server starting.
+        if (migrationInspector->isBlockingServer() and backend->hardFetchLedgerRangeNoThrow()) {
```

Comment:
I think the word `server` makes a reader think about web server. I would change it to `Clio`.